runtime/src/main/kotlin/io/github/seggan/myxal/runtime/RuntimeMethods.kt

```diff
@@ -31,20 +31,14 @@ private val LONG_MIN_VALUE_AS_BIG: BigInteger by lazy(LazyThreadSafetyMode.NONE)
 
 private val regexCache = mutableMapOf<String, Regex>()
 
-fun add(stack: ProgramStack): Any {
-    val b = stack.pop()
-    val a = stack.pop()
-    return addImpl(a, b)
-}
-
-private fun addImpl(a: Any, b: Any): Any {
+fun add(a: Any, b: Any): Any {
```

Comment:
I suggest swap `a` and `b` and remove that `SWAP` opcode in the compiler. Makes optimization... easier